config/production.json

```diff
@@ -1,6 +1,8 @@
 {
   "history": {
     "dataPath": "../cgus-data",
+    "snapshotsPath": "../cgus-data-snapshots",
+    "versionsPath": "../cgus-data",
```

Comment:
> Why not use this opportunity to unify production with defaults? 🙂

Reply:
In production I prefer having the three repositories at the same level but I have not impactful arguments…

Reply:
No worries, I'm happy with the current option if it feels better to you, I just like to have as little difference as possible between dev and production :wink: